### DISCUSSION TITLE
How to install a local server?

### DISCUSSION THREAD

**Discussion by mdre** [ArcadeData/arcadedb#103] [General]
Where can I found a compiled version of ArcadeDB?
I want to run it locally but out of docker.

**Comment by lvca** [comment]
You can download a release from GitHub:

https://github.com/ArcadeData/arcadedb/releases

**Reply by mdre** [reply]
Thank's Luca!
Startig the work to migrate our driver from ODBOGM to ADBOGM

**Reply by lvca** [reply]
Cool, please let me know how it's going.
